lib/settings/persistence/memory_settings_persistence.dart

```diff
@@ -0,0 +1,23 @@
+import 'settings_persistence.dart';
+
+class MemoryOnlySettingsPersistence implements SettingsPersistence {
+  bool musicOn = true;
+
+  bool soundsOn = true;
+
+  bool audioOn = true;
```

Comment:
I am really sorry, but I don't get an idea of having public properties in `SettingsPersistence` and also having an async methods which returns this properties. Could you please elaborate on that?

Reply:
Removed this file at all